src/main/java/com/google/sps/data/AvailabilityTimeSlotGenerator.java

```diff
@@ -180,9 +180,10 @@ private static Instant getStartOrEndOfWeek(ZonedDateTime day, boolean start) {
       if (hour > 12) {
         standardHour = hour - 12;
       }
-      String formattedTime =
-          String.format("%d:%02d %s", standardHour, minute, hour < 12 ? "AM" : "PM");
-
+      String formattedTime = "";
+      if (minute == 0) {
```

Comment:
Comment here on why this happens.

Reply:
I reverted to handling everything on the JSP page (this code no longer exists).